draft-ietf-httpbis-cache-latest.xml

```diff
@@ -1480,6 +1484,15 @@
 </t>
 </section>
 
+<section title="must-understand" anchor="cache-response-directive.must-understand">
+   <iref item="must-understand (cache directive)" primary="true" />
+<t>
+   The "must-understand" response directive indicates that a cache &MUST; 
+   understand and implement the caching semantics of the response to be able
+   to store it; if it does not, it cannot store the response.
+</t>
```

Comment:
Does this apply to just the status code, or to more elements of the response?

Reply:
I think it applies to the response status code, and whatever else it requires. Make sense? If so, will clarify.

Reply:
...which makes me think that the cache control directive should be called "must-understand-status"...